## vimplate_debian-path.diff by Stefano Zacchiroli <zack@debian.org>

Index: vim-scripts/plugin/vimplate.vim
===================================================================
--- vim-scripts.orig/plugin/vimplate.vim
+++ vim-scripts/plugin/vimplate.vim
@@ -15,6 +15,10 @@
 endif
 let loaded_vimplate = 1
 
+if !exists("Vimplate")
+  " Debian specific path unless overridden by the user
+  let Vimplate = "/usr/bin/vimplate"
+endif
 let s:vimplate = Vimplate
 
 function s:RunVimplate(template)
